ob1k-cache/src/main/java/com/outbrain/ob1k/cache/memcache/folsom/MemcachedClient.java

```diff
@@ -160,6 +174,7 @@ private boolean isOK(final MemcacheStatus memcacheStatus) {
             consumer.consume(Try.fromError(e));
           } catch (final ExecutionException e) {
             final Throwable error = e.getCause() != null ? e.getCause() : e;
+            log.error(cacheName + " " + error.getMessage(), error);
```

Comment:
try to be more informative:

"failed to run operation for key `{$key}`, error: `${errorMessage}`